style.css

```diff
@@ -49,6 +77,80 @@ main {
     overflow-x: hidden !important;
   }
 
+  .project_details {
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    margin: auto;
+    width: 90%;
+    height: 95%;
```

Comment:
- [ ] Well done on making the modal responsive for the mobile screen but kindly fix the design because all content of the modal should fit the screen.

Here you have a screenshot of your modal:

![image](https://user-images.githubusercontent.com/74506933/156667968-6627bda9-bc98-456f-8ab1-a55a4494a114.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/156668019-1552a7e8-35ee-4501-950a-e3869c63d0ad.png)